devicemodel/hw/pci/xhci.c

```diff
@@ -2627,6 +2627,10 @@ pci_xhci_complete_commands(struct pci_xhci_vdev *xdev)
 	trb = xdev->opregs.cr_p;
 	ccs = xdev->opregs.crcr & XHCI_CRCR_LO_RCS;
 	crcr = xdev->opregs.crcr & ~0xF;
+	if (!trb) {
+		UPRINTF(LDBG, "Get the invalid guest address!\r\n");
+		goto out;
```

Comment:
Our [coding guidelines prohibit use of goto](https://projectacrn.github.io/latest/developer-guides/c_coding_guidelines.html#c-st-08-goto-statement-shall-not-be-used)!  @NanlinXie